deisctl/deisctl.go

```diff
@@ -95,6 +96,8 @@ Options:
 	switch command {
 	case "list":
 		err = c.List(argv)
+  case "list-machines":
+    err = c.ListMachines(argv)
```

Comment:
This needs `gofmt` run on the source files--looks like Jenkins complained about this whitespace difference. :white_circle: